net/dnsviz/Portfile

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+# $Id$
+
+PortSystem         1.0
+
+PortGroup          github 1.0
+PortGroup          python 1.0
+
+github.setup       dnsviz dnsviz 0.6.5 v
+github.tarball_from releases
+
+categories         net
+platforms          darwin
+license            GPL-2+
+supported_archs    noarch
+maintainers        deccio.net:casey
+
+description        DNS measurement and diagnostic tool suite
+long_description \
+  DNSViz is a tool suite for measurement, diagnostic, and visualization of \
+  Domain Name System (DNS) behavior, including its security extensions \
+  (DNSSEC).
+
+checksums          rmd160 16a8b7094670e8c0012bbe362f52f6fb9828a3f9 \
+                   sha256 4598476625b2cd81224bc908d6fb98086c9568460d2074e35320fd651dadb269
+
+depends_lib        port:py27-dnspython \
+                   port:py27-m2crypto \
+                   port:py27-pygraphviz \
+                   port:bind9
+
+depends_build      port:bin/dot:graphviz
+
+python.default_version    27
+destroot.destdir  --prefix=${python.prefix} --root=${destroot} --install-data=${prefix} --install-scripts=${prefix}/bin/
```

Comment:
It looks like I can remove `--install-scripts` and the `python.link_binaries` (default) does its job by linking properly to the script.

However, without `--install-data` option all the data goes in `/opt/local/Library/Frameworks/Python.framework/Versions/2.7/`.  The biggest concerns are the man pages, but secondarily it would be nice to have access to `share/dnsviz/` and `share/doc/dnsviz/` in `/usr/local`.